src/unstract/sdk/embedding.py

```diff
@@ -79,9 +88,29 @@ def _get_embedding_length(self) -> int:
         embedding_dimension = len(embedding_list)
         return embedding_dimension
 
-    @deprecated("Use the new class Embedding")
+    def get_class_name(self) -> str:
+        """Gets the class name of the Llama Index Embedding.
+
+        Args:
+            NA
+
+            Returns:
+                Class name
+        """
+        return self._embedding_instance.class_name()
+
+    @deprecated("Deprecated class and method. Use Embedding instead of ToolEmbedding")
```

Comment:
@gaya3-zipstack `deprecated` library will throw warning that it is deprecated, we do not need to mention again.

In that case let's change message to `Use Embedding instead of ToolEmbedding.`.